File: app/src/main/java/mobi/porquenao/poc/kotlin/core/AppDatabase.java
package mobi.porquenao.poc.kotlin.core;

import com.raizlabs.android.dbflow.annotation.Database;

@Database(name = AppDatabase.NAME, version = AppDatabase.VERSION, generatedClassSeparator = "_")
public class AppDatabase {

    public static final String NAME = "app";
    public static final int VERSION = 1;

}

File: app/src/main/java/mobi/porquenao/poc/kotlin/core/AppDatabase.kt
package mobi.porquenao.poc.kotlin.core

import com.raizlabs.android.dbflow.annotation.Database

Database(name = AppDatabase.NAME, version = AppDatabase.VERSION, generatedClassSeparator = "_")
public object AppDatabase {
    public val NAME: String = "app"
    public val VERSION: Int = 1
}
